### PR TITLE
Add shop page.

### PR DESCRIPTION
**Pull request by olhavalchak**

### CODE REVIEW

File: src/theme/styles/shop.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  color: white;
+  width: 100px;
+  height: 20px;
+}
+@media screen and (max-width: 960px) {
+  .items-container,
+  .items-list-container {
+    margin-top: 0;
+  }
+}
```
Comment (olhavalchak) [inline src/theme/styles/shop.css:92]: I'll fix this in next commit:)

File: src/constants/shop.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+export const SHOP_ITEMS = [
+  {
+    id: 1,
+    img: 'https://interactive-examples.mdn.mozilla.net/media/cc0-images/grapefruit-slice-332-332.jpg',
+    title: 'fruit',
+    desc: 'Very delicious fruit, juicy',
```
Comment (AndreiYar) [inline src/constants/shop.js:6]: Rename to "description" please.

File: src/pages/shop/Shop.jsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { SHOP_ITEMS } from '../../constants';
+
+export const Shop = () => (
+  <div className="shop-container">
+    <div className="items-list-container">
+      <ul className="items-list-parent">
+        <li className="items-list">СИРОВАТКИ</li>
```
Comment (AndreiYar) [inline src/pages/shop/Shop.jsx:8]: Create array of items, please.

File: src/constants/shop.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+export const SHOP_ITEMS = [
+  {
+    id: 1,
+    img: 'https://interactive-examples.mdn.mozilla.net/media/cc0-images/grapefruit-slice-332-332.jpg',
+    title: 'fruit',
+    description: 'Very delicious fruit, juicy',
+    price: '100$',
+    button: 'Add to card',
```
Comment (AndreiYar) [inline src/constants/shop.js:8]: Remove button label

File: src/theme/styles/index.css [hunk 5]
```diff
@@ -5,3 +5,4 @@
 @import './aboutMe.css';
 @import './skinTest.css';
 @import '../../../node_modules/rc-steps/assets/index.css';
```
Comment (AndreiYar) [inline src/theme/styles/index.css:7]: Move to the first place, pls

File: src/theme/styles/shop.css [hunk 6]
```diff
@@ -0,0 +1,92 @@
+.shop-container {
+  height: 100vh;
+  width: 100vw;
+  background: rgb(92, 75, 75);
```
Comment (AndreiYar) [inline src/theme/styles/shop.css:4]: Convert all rgb to hex format.

File: src/theme/styles/shop.css [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+  height: 100%;
+  background: rgb(247, 245, 245);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+.items-list {
+  border-radius: 2px;
+  padding: 10px;
+ padding-left: 15px;
```
Comment (AndreiYar) [inline src/theme/styles/shop.css:19]: Add space from the left.

File: src/theme/styles/shop.css [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+  background: rgb(141, 128, 128);
+}
+.explanation-container {
+  width: 50%;
+}
+.res-points-container,
+.res-explanation-container {
+  display: flex;
+  justify-content: space-around;
+
```
Comment (AndreiYar) [inline src/theme/styles/shop.css:46]: Remove empty line.

File: src/pages/shop/Shop.jsx [hunk 9]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { SHOP_ITEMS, SECTIONS } from '../../constants';
+// const { section } = SHOP_ITEMS.find((item) => item.section === SECTIONS.title);
```
Comment (AndreiYar) [inline src/pages/shop/Shop.jsx:3]: Remove the comment.

**Pull request merged**